community/php7/APKBUILD

```diff
@@ -538,6 +545,7 @@ common() {
 	pkgdesc="$pkgdesc (common config)"
 	provides="$pkgname-zlib"  # for backward compatibility
 	depends=""
+	provides="php"
```

Comment:
this looks overrides `provides="$pkgname-zlib"` 2 lines above https://wiki.alpinelinux.org/wiki/APKBUILD_Reference#provides

Reply:
That was done in https://github.com/alpinelinux/aports/pull/2951